src/zc/buildout/buildout.txt

```diff
@@ -1264,10 +1264,10 @@ Expressions must not contain either the ``#`` or the ``;`` character.
 Adding and removing options
 ---------------------------
 
-We can append and remove values to an option by using the + and -
+We can append and remove values to an option by using the ``+`` and ``-``
 operators.
 
-This is illustrated below; first we define a base configuration.
+This is illustrated below; first we define a base configuration::
```

Comment:
> http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#doctest-blocks

Oh! I wasn't aware that this is part of ReST. I managed to miss that
part of the docs. So Vim's syntax file should be fixed.

> (Thinking about ReST while editing in MD makes me sad. Silly GitHub.)

Agreed completely ...

-- 
jean                                              . .. .... //\\\oo///\\